x-pack/plugins/lens/kibana.json

```diff
@@ -20,6 +20,7 @@
     "share",
     "presentationUtil",
     "dataViewFieldEditor",
+    "dataViewEditor",
```

Comment:
It seems like this is only used for the permissions, is there any difference between `userPermissions` of `dataViewEditor` and `dataViewFieldEditor`? The implementation looks identical

Reply:
It is also used for opening the dataView editor which is not the same. From my understanding these are two different editors with different permissions.

Reply:
A right, makes sense